src/Utils/GalleryUtils.ts

```diff
@@ -223,11 +224,15 @@ export function downloadItem(
 
   const name = data.name;
   useDialog.getState().showOkCancelModalDialog(
-    "Download to My Notebooks",
-    `Download ${name} from gallery as a copy to your notebooks to run and/or edit the notebook.`,
+    `Download to My Notebooks ${userContext.features.phoenix ? "Scratch" : ""}`,
```

Comment:
this is used repeatedly, have a helper to choose between "My Notebooks" and "My Notebooks Scratch"

Reply:
addressed